server/http_status.go

```diff
@@ -193,6 +193,35 @@ func (s *Server) startHTTPServer() {
 	serverMux.HandleFunc("/debug/pprof/profile", tracecpu.ProfileHTTPHandler)
 	serverMux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 	serverMux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+	ballast := make([]byte, 0)
+	serverMux.HandleFunc("/debug/ballast-object-sz", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
```suggestion
	serverMux.HandleFunc("/debug/ballast", func(w http.ResponseWriter, r *http.Request) {
```

Reply:
Maybe the pathname "/debug/ballast-object-sz" is better than "/debug/ballast", because the previous one is more descriptive? For example, when your GET or POST on the path "/debug/ballast-object-sz", the pathname tells you everything that would happen.